tests/client_test.go

```diff
@@ -0,0 +1,23 @@
+package easypost_test
+
+import (
+	"github.com/EasyPost/easypost-go"
+)
+
+func (c *ClientTests) TestClientTimeout() {
+	timeout1 := 1000
+	timeout2 := 10
+	tempClient := c.TestClient()
+	// set the timeout as part of the client initialization
+	client := &easypost.Client{
+		APIKey:  tempClient.APIKey,
+		Timeout: timeout1,
+	}
+	assert := c.Assert()
+	// test that property has been set correctly in the EasyPost.Client instance
+	assert.Equal(client.Timeout, timeout1)
+	// override the Timeout property after initialization
+	client.Timeout = timeout2
+	// test that property has been changed correctly
+	assert.Equal(client.Timeout, timeout2)
```

Comment:
I think these two **assert.Equal** parameters are in the wrong order. It should be **expected, actual.**

Reply:
Resolved in [0e54b0d](https://github.com/EasyPost/easypost-go/pull/47/commits/0e54b0da03341f2519e43e28a721b507eb295f53)